lms/services/course.py

```diff
@@ -115,14 +113,6 @@ def _search_query(  # noqa: PLR0913, PLR0917
                 .filter(Organization.id.in_(organization_ids))
             )
 
-        if h_userids:
```

Comment:
_search_query was being used by the dashboard query below just passing this, which was added for the dashboards.

Moving it to the dashboard query below.